conf.py

```diff
@@ -174,57 +174,67 @@
 # -- Options for LaTeX output --------------------------------------------------
 
 # The paper size ('letter' or 'a4').
-#latex_paper_size = 'letter'
+# latex_paper_size = 'letter'
 
 # The font size ('10pt', '11pt' or '12pt').
-#latex_font_size = '10pt'
+# latex_font_size = '10pt'
 
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, documentclass [howto/manual]).
 latex_documents = [
-  ('index', 'PythonDevelopersGuide.tex', u'Python Developer\'s Guide Documentation',
-   u'Brett Cannon', 'manual'),
+    (
+        'index',
+        'PythonDevelopersGuide.tex',
+        u'Python Developer\'s Guide Documentation',
+        u'Brett Cannon',
+        'manual',
+    )
```

Comment:
Trailing comma makes the list easier for modification.

And I'm not sure that writing 5-tuple vertically adds readability (especially if add yet few 5-tuples) .